include/z64environment.h

```diff
@@ -72,26 +136,26 @@ typedef struct {
     /* 0x14 */ char unk_14[0x01];
     /* 0x15 */ u8 skyboxDisabled;
     /* 0x16 */ u8 sunMoonDisabled;
-    /* 0x17 */ u8 unk_17; // currentWeatherMode for skybox? (prev called gloomySky)
-    /* 0x18 */ u8 unk_18; // nextWeatherMode for skybox?
-    /* 0x19 */ u8 unk_19;
-    /* 0x1A */ u16 unk_1A;
+    /* 0x17 */ u8 skyboxConfig; // only used outdoors
+    /* 0x18 */ u8 nextSkyboxConfig; // only used outdoors
+    /* 0x19 */ u8 changeSkyState;
+    /* 0x1A */ u16 changeSkyTimer;
```

Comment:
I think it's not obvious enough the "change" stuff is about changing to the "next" configs, here's an idea
```suggestion
    /* 0x18 */ u8 changeSkyboxNextConfig; // only used outdoors
    /* 0x19 */ u8 changeSkyboxState;
    /* 0x1A */ u16 changeSkyboxTimer;
```
see also next comment